doc/Repeater.md

```diff
@@ -1,8 +1,63 @@
+# Quantum Repeater
 
-*Note: this file is an early design document, but might be updated as the version goes up.*
+## What is this?
 
-# Repeater #
+Quantum Repeater is one crucial network device in entanglement-based quantum networking, which allows us to extend link-level entanglement to end-to-end entanglement. 
 
-A Repeater node has exactly two external links, so that it is useful in a line only.  It is essentially a repurposed [QNIC](QNIC), with both halves of the card running the same [Network Type](Network Type). It also contains a simple [C4](C4).
+The main works of quantum repeaters are following.
 
-A repeater does not need to run a full [Routing](Routing) protocol, since it has simple decisions on what the next hop is.
\ No newline at end of file
+1. Creating base entanglement
+2. Entanglement extension
+3. Error management
+4. Network operations
+
+### 1. Creating base entanglement
+
+Quantum Repeater generates entanglement between neighbor nodes known as link-level entanglement, relatively shorter distance entanglement. In practice, the distances between neighbor nodes should be decided by characteristics of the channel, such as attenuation, loss, error. 
+
+![Repeater.drawio (1).png](./img/Repeater.png)
+
+Several link types have been proposed to generate this link-level entanglement, such as Memory-Memory architecture, Memory-Interfere-Memory architecture.  
+
+### 2. Entanglement extension
+
+The entanglement swapping is performed in each repeater station and extends link-level entanglement to end-to-end entanglement. Repeaters perform Bell state measurement to pair of qubits and extend link-level entanglement to multihop entanglement.
+
+![EntanglementSwapping.drawio (2).png](./img/EntanglementSwapping.png)
+
+At the end of this process, the repeater must tell the result of Bell measurement to proper partners to correct state with Pauli operations. 
+
+### 3. Error management
+
+Quantum error is one critical obstacle for large-scale quantum networking. Keeping the fidelity of the quantum state is also one important task of a quantum repeater. For the near future (called 1G quantum networking), entanglement purification plays an important role to find errors in the quantum state and improving its quality.
+
+### 4. Network operations
+
+The repeater node behaves as an ordinal network node that is responsible for routing, multiplexing, etc. As mentioned above, repeaters must be capable of coordinating operations with other repeaters or end nodes.
```

Comment:
Here, a Repeater is said to participate in routing, which is only true in a limited way for two-interface Repeaters.

Reply:
OK, I'll fix it.